### ISSUE TITLE
injectView should have view.detach();

### ISSUE THREAD

**Issue by arikwex** [vecnatechnologies/backbone-torso#32]
Calling inject view in succession should work...
to make that work, we must internally call detach.
